src/index.js

```diff
@@ -1 +1,37 @@
-import './styles.css';
\ No newline at end of file
+import './styles.css';
+import {
+  name,
+  score,
+  submitBtn,
+  refreshBtn,
+} from './utils.js';
+import sendData from './sendData.js';
+
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/JBpxyjlHr5ZjiBM74biA/scores/';
+
+submitBtn.addEventListener('click', async () => {
+  // eslint-disable-next-line no-restricted-globals
```

Comment:
- [OPTIONAL]  `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.